docs/user/use_pretrained_network_to_start_new_experiment.md

```diff
@@ -0,0 +1,79 @@
+# Use a pre-trained network to start a new experiment
+
+In this section you will learn how to load a pre-trained network and use it in
+new experiments. We'll cover two cases in particular:
+
+- How to use a trained policy as an expert in Behavioral Cloning
+- How to reuse a trained Q function in DQN
+
+Before attempting either of these, you'll need a saved experiment snapshot. [This page](save_load_resume_exp.md)
+will show you how to get one.
+
+## Using a pre-trained policy as a BC expert
+
+There are two steps involved. First, we must load the pre-trained policy. Assuming
+that it was trained with garage, details on extracting a policy from a saved experiment
+can be found [here](reuse_garage_policy.md). Next, we setup a new experiment and
+pass the policy as the `source` argument of the `BC` constructor:
+
+```python
+# Load the policy
+from garage.experiment import Snapshotter
+snapshotter = Snapshotter()
+snapshot = snapshotter.load('path/to/snapshot/dir')
+
+expert = snapshot['algo'].policy
+env = snapshot['env']  # We assume env is the same
+
+# Setup new experiment
+from garage import wrap_experiment
+from garage.experiment import LocalRunner
+from garage.torch.algos import BC
+from garage.torch.policies import GaussianMLPPolicy
+
+@wrap_experiment
+def bc_with_pretrained_expert(ctxt=None):
+    runner = LocalRunner(ctxt)
+    policy = GaussianMLPPolicy(env.spec, [8, 8])
+    batch_size = 1000
+    algo = BC(env.spec,
+              policy,
+              batch_size=batch_size,
+              source=expert,
+              max_path_length=200,
+              policy_lr=1e-2,
+              loss='log_prob')
+    runner.setup(algo, env)
+    runner.train(100, batch_size=batch_size)
+
+
+bc_with_pretrained_expert()
+```
+
+Please refer to [this page](algo_bc.md) for more information on garage's implementation
```

Comment:
Same for this. Better to link to actual garage doc page instead of the md file on github.